binding/richclient.go

```diff
@@ -0,0 +1,117 @@
+package binding
+
+import (
+	"fmt"
+
+	"github.com/konveyor/controller/pkg/logging"
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/settings"
+)
+
+var (
+	Settings = &settings.Settings
+	Log      = logging.WithName("binding")
+)
+
+var (
+	// Addon An addon richClient configured for a task execution.
+	RichClient *Adapter
+	// Client
+	//client *Client
+)
+
+func init() {
+	err := Settings.Load()
+	if err != nil {
+		panic(err)
+	}
+
+	RichClient = newRichClient()
+}
```

Comment:
I think we decided to not provide the `RichClient` global.
Please remove the `var` and the `init()`.
Please rename `Adapter` => `RichClient`.

Reply:
:+1: Updated (the init() was left for Settings load)